opencv/platform/pom.xml

```diff
@@ -90,6 +90,12 @@
       <version>${project.version}</version>
       <classifier>${javacpp.platform.linux-arm64}</classifier>
     </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>${javacpp.moduleId}</artifactId>
+      <version>${project.version}</version>
+      <classifier>${javacpp.platform.linux-loongarch64}</classifier>
+    </dependency>
```

Comment:
We'll need to keep these entries in all platform/pom.xml commented out until we're able to get builds running as part of CI. You can leave them there, but just in comments for now as per other ones in the files.